mage_integrations/mage_integrations/destinations/doris/__init__.py

```diff
@@ -0,0 +1,201 @@
+from typing import Dict, List, Tuple
+
+from mage_integrations.connections.mysql import ConnectionMethod
```

Comment:
are these imports correct? you want to import from `connections.doris` `destinations.doris.utils` right?

Reply:
Actually, doris is the same as mysql, I will change to doris.